src/components/ToggleButtonGroup/styles.ts

```diff
@@ -0,0 +1,23 @@
+import styled from "styled-components";
+import { ToggleButtonGroup, ToggleButton } from "react-aria-components";
+import { colors } from "../../theme";
+
+export const StyledToggleButtonGroup = styled(ToggleButtonGroup)`
+    display: flex;
+`;
+
+export const StyledToggleButton = styled(ToggleButton)`
+    min-width: 4.4rem;
+    height: 3.6rem;
+    padding: 0.8rem;
+    background-color: ${colors.palette.white};
+    border: solid 0.1rem ${colors.palette.pale};
```

Comment:
Tiny thing I didn't notice before: can we do a `& + &` rule here to remove the left border?